### PR TITLE
Sils/084

### PR DESCRIPTION
**Pull request by sils1297**

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
ack 98a0003

File: package.json [hunk 1]
```diff
@@ -4,12 +4,14 @@
   "dependencies": {
     "alex": "~2",
     "autoprefixer": "~6",
+    "babel-eslint": "~6",
```
Comment (Adrianzatreanu) [inline package.json]: we want these 2 additions to NpmRequirements from REQUIREMENTS too :)

**Comment by Adrianzatreanu** [comment]
unack fccf7ff107a133695633bf7dd2c1e254002e8882

File: RELEASE_NOTES.rst [hunk 2]
```diff
@@ -1,3 +1,13 @@
+coala-bears 0.8.4
+=================
+
+This bugfix release fixes the following issues:
```
Comment (Adrianzatreanu) [inline RELEASE_NOTES.rst]: youre saying: "the following issues" but what youre listing is things that are now working well.

File: package.json (hunk 1 above)
Comment (sils1297) [inline package.json]: they are not needed in the standard usecase but in a special one where eslint needs additional deps, we're using that in tests so it's not in here. If you feel this is wrong I'd file an issue as this commit is already in master.

**Comment by Adrianzatreanu** [comment]
ack 59646ae7433d335ca3feace8d2122329c2ec4043

**Comment by Adrianzatreanu** [comment]
ack fccf7ff107a133695633bf7dd2c1e254002e8882

**Comment by Adrianzatreanu** [comment]
if it doesnt break normal usage then its fine.

**Comment by Adrianzatreanu** [comment]
ack 4a119ef44aecf81e8cd0a916543c8ed1c0a468f7

**Comment by Adrianzatreanu** [comment]
ack fe4958f49c6aabd50afcabc0a747090f7a6a2ae4

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**